sdk/resourcemanager/azure-resourcemanager-resources/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "java",
+  "TagPrefix": "java/resourcemanager/azure-resourcemanager-resources",
+  "Tag": "java/resourcemanager/azure-resourcemanager-resources_da922b0432"
```

Comment:
BTW, api-version still there?
https://github.com/Azure/azure-sdk-assets/blob/java/resourcemanager/azure-resourcemanager-resources_da922b0432/java/sdk/resourcemanager/azure-resourcemanager-resources/src/test/resources/session-records/DeploymentsTests.canBeginCreateAsync.json#L4

Reply:
And `retry-after` seems not handled.
https://github.com/Azure/azure-sdk-assets/blob/java/resourcemanager/azure-resourcemanager-resources_da922b0432/java/sdk/resourcemanager/azure-resourcemanager-resources/src/test/resources/session-records/DeploymentsTests.canDeployVirtualNetwork.json#L741

Wondering whether playback can finish in 30 seconds (current cut-off time).

Reply:
api-version still there is expected, but in playback mode, we will ignore match api-version, the logic is here: https://github.com/Azure/azure-sdk-for-java/blob/906806fe0b78f7716973f65062dfcb3d6f9db556/sdk/resourcemanager/azure-resourcemanager-test/src/main/java/com/azure/resourcemanager/test/ResourceManagerTestProxyTestBase.java#L324